docs/operators/transform.md

```diff
@@ -0,0 +1,167 @@
+Transform - Operator Documentation
+==================================
+
+About Transform operator
+------------------------
+
+Transform means mapping of field expression from input to output or conversion of fields from one type to another.
+This operator is stateless. This operator receives objects on its input port; for each such input object, it creates a new output object whose fields are computed as expressions involving fields of the input object. 
+The types of the input and output objects are configurable as are the expressions used to compute the output fields. 
+
+The operator class is `TransformOperator` located in the package `com.datatorrent.lib.transform`.
+
+Use Case
```

Comment:
Can you make this title type similar to "Configuration Parameter" i.e. with ####